New VGBC Style Overlay

JinchanJ opened the pull request:
Hi, this is the new VGBC style overlay. Thanks!

joaorb64 commented:
Awesome! Can I rename it to something like... vgbootcampy or something like that? lol

JinchanJ commented:
Yeah, you can rename it to vgbootcampy or something.

JinchanJ commented:
Renamed it to scoreboard_vgbootcampy

JinchanJ commented:
One sec, I think I found a bug. I think it'll be a quick fix.

JinchanJ commented:
I fixed the bug where the previous overlay elements remained when the program restarted. Now they disappear when the program restarts. Also renamed ssbultimate.html to index.html.

JinchanJ commented:
I think it's done for now. Appreciate it if it can be checked. Thanks!

JinchanJ commented:
Actually, adding a small box in the top left to display tournament name. Almost done.

JinchanJ commented:
I think it's done now. Thanks!

JinchanJ commented:
Most likely the last update. The Twitter is replaced with the player's name when the player do not have a Twitter.

joaorb64 commented:
Such attention to detail, great job! Merging! :)

JinchanJ commented:
Thank you!
